### PR TITLE
New package: ibus-avro-1.1

### PR DESCRIPTION
**Pull request by mnabid**
Avro Phonetic Bangla Layout for IBus.

### CODE REVIEW

**Comment by mnabid** [comment]
`i686` build failed due to missing shlib `libffi.so.6` (looks like issue #20336 isn't solved yet for cross builds). 
But why **libffi-3.3.tar.gz** can't be fetched in `aarch64` and `aarch64-musl` builds? I can fetch it manually with no issues. Any help?

File: srcpkgs/ibus-avro/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'ibus-avro'
+pkgname=ibus-avro
+version=1.1
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
```
Comment (xtraeme) [inline srcpkgs/ibus-avro/template]: this is set by default, you can remove this line

File: srcpkgs/ibus-avro/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+distfiles="https://github.com/sarim/${pkgname}/archive/${version}.tar.gz"
+checksum=402117a38d5d646f73775eb9ca33a685245caf897f3ecf04f845ac1789988fd0
+
+pre_configure() {
+	autoreconf -vfi
+}
+
+post_install() {
+	vlicense MPL-1.1.txt COPYING
+}
```
Comment (xtraeme) [inline srcpkgs/ibus-avro/template]: MPL-1.1 is part of `base-files`, so that you don't need to install it.

**Comment by mnabid** [comment]
Done.

File: srcpkgs/ibus-avro/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'ibus-avro'
+pkgname=ibus-avro
+version=1.1
+revision=1
+build_style=gnu-configure
```
Comment (xtraeme) [inline srcpkgs/ibus-avro/template]: The package does not install any binary, so this needs `archs=noarch`

**Comment by mnabid** [comment]
@xtraeme

**Pull request merged**